tutorials/setup-your-development-environment.md

```diff
@@ -0,0 +1,67 @@
+# Setup your development environment
+
+The following section walks you through the necessary steps to setup your environment.
+It is recommended to spin up a clean Ubuntu VM for these steps as it is always good to perform experiments in an isolated development environment.
+The steps outlined [here](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass) explain how to do this.
```

Comment:
```suggestion
Follow the steps in [Set up an Ubuntu VM with Multipass](https://juju.is/docs/sdk/set-up-your-development-environment#heading--set-up-an-ubuntu-vm-with-multipass) to do this.
```
"Here" is a very bad link text, since users have no idea where they'll go, and Google doesn't like it either.

Reply:
Cool - thanks for the explanation. Especially, that Google doesn't like it was new to me.